docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+$ cd tools
+$ make tfuser
+cd tfuser && go build -ldflags '-w -s -X github.com/threefoldtech/zos/pkg/version.Branch=fcvm -X github.com/threefoldtech/zos/pkg/version.Revision=c00740b8fbb75746cd02fb5614ccf57342b7a9f4 -X github.com/threefoldtech/zos/pkg/version.Dirty='
+$ cd tfuser
+$ ./tfuser --help
+```
+
+## Identity on the TF Grid
+
+To interact with the grid we need an indentity which is simply some cryptographic keys that we will use to sign transaction on the network.
+Let's generate them with tfuser
+
+```
+$ tfuser id
+new identity generated: 7HZd49ugHzJxQUnHKyQRbmcwWWvKKtoJUndfiiXGTkow
+seed saved at user.seed
+```
+
+NB: a file named user.seed has been created. This file is sensitive as it stores your private key. Make sure not to loose that file.
+
+## Provisioning
+
+To provision our VMs we first need to setup a network between different Nodes on the TFGrid
+We will use devnet Nodes to select some nodes go to :[devnet Cockpit](https://cockpit.devnet.grid.tf/)
+We picked three nodes identified by these NodeId
+
+- nodeid1: FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242
+- nodeid2: 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm
+- nodeid3: 2anfZwrzskXiUHPLTqH1veJAia8G6rW2eFLy5YFMa1MP
+
+Becarefull NodeId is case sensitive
+
+### Networking
+
+We now need to create a network between the nodes. Hopefully tfuser can help us generate the configuration file.
+Let's first create a network named `kubetest` with a large cidr `173.30.0.0/16` and write that configuration inside a file `network.json`
+
+```
+$ ./tfuser generate --schema network.json network create --name kubetest --cidr 173.30.0.0/16
+```
+
+now let's add the nodes that we have selected inside our network by specifying the path to our `network.json` network configuration file. We also specify a port for wireguard to use and a subnet to use on the nodes.
+
+```
+$ ./tfuser generate --schema network.json network add-node --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3  \
+				      --subnet 173.30.1.0/24
+$ ./tfuser generate --schema network.json network add-node --node 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm \
+				      --subnet 173.30.2.0/24
+$ ./tfuser generate --schema network.json network add-node --node FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242 \
+				      --subnet 173.30.3.0/24
+```
+
+there are two levels here :
+
+- one for the global network with a cidr /16
+- one for network resources on the node with a cidr /24. All workloads in this network resource will receive an IP from this subnet. The subnet must be part of the CIDR provided when the network is created.
+
+**NOTE** that you can't give the same subnet to all of your nodes. Each node must be assigned a different /24 subnet that is part of the global /16 of the network.
+
+We will need an external access to the network to be able to ssh into our VMs.
+
+```
+$ ./tfuser generate --schema network.json network add-access --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3 \
```

Comment:
Maybe a not regarding how to pick an entry point node here.
If user doesn't have IPv6, then they have to make sure to pick a node that has a public IPv4 address and use the `--ip4` flag.
For user with IPv6, the `--ip4` flag is not required

Reply:
probably but for now the only one working for an external access was node : qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3.

 I would love to try with a node that has a pulbic IPv6 address . Do you know a node that can work  @zaibon ?

Reply:
> I would love to try with a node that has a pulbic IPv6 address . Do you know a node that can work @zaibon ?

All the nodes have ipv6. So you should be able to pick any.